homeassistant/components/zwave/node_entity.py

```diff
@@ -229,3 +239,10 @@ def device_state_attributes(self):
             attrs[ATTR_WAKEUP] = self.wakeup_interval
 
         return attrs
+
+    def _compute_unique_id(self):
+        if self._manufacturer_name and self._product_name:
+            return '{}-{}-{}'.format(self.node_id,
```

Comment:
Can't we just use `self.node_id` now ?

Reply:
You mean wait for manufacturer & product and then use only node_id?
We could, but I think that if we have the extra data using it will make the id more unique and more readable for manual edits.

Reply:
I thin it is better to use more info in any case. I don't feel strongly about this.

Reply:
I think that we should limit unique ID to the least amount of unique data that we can have. btw, should we incorporate values too ?